packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -72,7 +62,7 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
     shallow,
   );
 
-  const { canChangePasswordWithOldPassword, canExportAccountData, canDeleteAccount } = accountController.getFeatures();
+  const { canChangePasswordWithOldPassword } = accountController.getFeatures();
```

Comment:
This mechanism is in place to disable these features for specific integrations. Why would you remove this mechanism and implementation?

Just set `canExportAccountData` and `canDeleteAccount` to false and done, right? Custom integrations can still implement these features.